samples/SwiftDemoApp/SwiftDemoApp/AppDelegate.swift

```diff
@@ -17,7 +17,7 @@ import GoogleMaps
 import UIKit
 
 // Change this key to a valid key registered with the demo app bundle id.
-let kMapsAPIKey = ""
+let kMapsAPIKey = "YOUR-API-KEY"
```

Comment:
We have a check down below for an empty key and print out an error message to the user. So please either revert this change or update the check.